docs/source/upcoming_release_notes/800-New_lamp_configuration_for_magnetic_bottle.rst

```diff
@@ -0,0 +1,30 @@
+800 New lamp configuration for magnetic bottle
+#################
+
+API Changes
+-----------
+- new LAMPMangeticBottle device class
```

Comment:
This isn't an API change- or if it is, I need to rename this category to "Backwards-incompatible API breaks". This should not slow down the merge.